PowerShell/Deploy/Setup-Dependencies.ps1

```diff
@@ -18,6 +18,9 @@ ForEach ($DependentModule In $DependentModules)
         if ($DependentModule -eq 'PowerShellGet'){
             Install-Module -Name $DependentModule -Repository:('PSGallery') -AllowPrerelease -RequiredVersion '3.0.0-beta10' -Force
         }
+        elseif ($DependentModule -eq 'PSScriptAnalyzer') {
+            Install-Module -Name $DependentModule -Repository:('PSGallery') -RequiredVersion '1.19.1' -Force
```

Comment:
New version of PSScriptAnalyzer does not play nice with our CI pipeline, need to investigate further